global/css/_global.css

```diff
@@ -219,6 +224,18 @@ svg {
   @apply su-mx-0 su-transition su-opacity-100 su-w-20 su-h-20 su-bg-transparent su-border-2 su-border-solid su-border-black-60;
 }
 
+.component-slider.su-slider-dark .component-slider-vertical-videos + .component-slider-controls {
+  @apply su-cc;
+}
+
+.component-slider.su-slider-dark .component-slider-vertical-videos + .component-slider-controls .swiper-pagination-bullet {
+  @apply su-border-2 su-border-white hocus:su-border-dark-mode-red;
+}
+
+.component-slider.su-slider-dark .component-slider-vertical-videos + .component-slider-controls .swiper-pagination-bullet.swiper-pagination-bullet-active {
+  @apply su-border-dark-mode-red su-bg-dark-mode-red;
+}
+
```

Comment:
The Vertical video section has a dark image overlay background, so we're making the carousel bullets border white which is more accessible than the grey that was used on plain black background (eg image gallery modal).